ehttp/src/streaming/mod.rs

```diff
@@ -0,0 +1,30 @@
+use std::ops::ControlFlow;
```

Comment:
Let's add a short `//! Module doc` here to guide users:

![image](https://github.com/emilk/ehttp/assets/1148717/1e97b04c-1bc0-4b82-9c64-e87d65929b25)

(`cargo doc --no-deps --all-features --open`)

Reply:
I added an example and a short description